scapy/modules/p0f.py

```diff
@@ -14,6 +14,7 @@
 import os
 import socket
 import random
+import scapy.modules.six as six
```

Comment:
You can write `from scapy.modules import six`. Also, since that's a Scapy import, please move it with the other Scapy imports.